packages/tables/src/Filters/Concerns/HasLabel.php

```diff
@@ -9,19 +9,32 @@ trait HasLabel
 {
     protected string | Closure | null $label = null;
 
+    protected bool $shouldTranslateLabel = false;
+
     public function label(string | Closure | null $label): static
     {
         $this->label = $label;
 
         return $this;
     }
 
+    public function translateLabel(bool $shouldTranslateLabel = true): static
+    {
+        $this->shouldTranslateLabel = $shouldTranslateLabel;
+
+        return $this;
+    }
+
     public function getLabel(): string
     {
-        return $this->evaluate($this->label) ?? (string) Str::of($this->getName())
+        $label = $this->evaluate($this->label) ?? (string) Str::of($this->getName())
             ->before('.')
             ->kebab()
             ->replace(['-', '_'], ' ')
             ->ucfirst();
+
+        return $this->shouldTranslateLabel
```

Comment:
Please move this conditional onto one line, then we are ready to merge

Reply:
done

Reply:
There are still places this needs to be fixed please

Reply:
those have a complex if condition so I usually split lines in those cases.
Sure you want them in a single line?

Reply:
Fair enough, ok